tokio-trace/src/level_filters.rs

```diff
@@ -0,0 +1,95 @@
+use std::cmp::Ordering;
+use tokio_trace_core::Level;
+
+/// Describes the level of verbosity of a span or event.
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+pub struct LevelFilter(Option<Level>);
+
+impl LevelFilter {
+    /// The "off" level.
+    ///
+    /// Designates to turn off logging.
```

Comment:
Nit: this wording seems a little awkward, maybe rephrase it as something like 
```suggestion
    /// Designates that logging should be to turned off.
```